src/main/java/uk/gov/hmcts/reform/sendletter/services/pdf/PdfCreator.java

```diff
@@ -43,6 +46,19 @@ public PdfCreator(DuplexPreparator duplexPreparator, IHtmlToPdfConverter convert
         return PdfMerger.mergeDocuments(docs);
     }
 
+    public byte[] createFromBase64PdfWithCopies(List<Doc> docs) {
+        Asserts.notNull(docs, "base64decodedDocs");
+
+        List<byte[]> pdfs = docs
+            .stream()
+            .map(d -> Collections.nCopies(d.copies, d.content))
+            .flatMap(Collection::stream)
+            .map(duplexPreparator::prepare)
```

Comment:
I think this we should do this before nCopies operations as this method creates new copies ???